invenio_rdm_records/theme/templates/invenio_rdm_records/details/list_entities.html

```diff
@@ -8,20 +8,20 @@
 
 {% macro list_entities(entities) %}
   {% for entity in entities %}
-    {%- if entity.identifiers and entity.identifiers[0] %}
-      {# TODO: Is the first identifier given precedence or is "Orcid" given precedence? #}
-      {# Assume first identifier is given precedence #}
-      {% set main_identifier = entity.identifiers[0] %}
-      {%- if main_identifier.scheme == "Orcid" %}
-      <a href="{{ main_identifier.identifier|pid_url('orcid') }}">
-        <img class="inline-orcid" src="{{ url_for('static', filename='images/orcid.svg') }}" />
-      </a>
-      {# TODO: Other potential main identifiers #}
-      {%- endif %}
-    {% else %}
-      {# TODO: Change for an organization icon when organizations supported #}
-      <i class="fa fa-user" aria-hidden="true"></i>
+    {# TODO: Is the first identifier given precedence or is "Orcid" given precedence? #}
```

Comment:
Shouldn't identifiers be a dict instead of list? Meaning we cannot select the first one? For now until this is fixed, selecting the first makes sense.

```
{
'orcid': …
'gnd': …
}
```

Reply:
I thought `identifiers` were an array of scheme and identifier (for every place identifiers appear in the schema) to match up with the deposit page where one adds another identifier to a list (reusable front-end component). But they could be adding to a dict in the case where each entry must be unique per scheme... @ppanero do you remember if we had a specific reason in mind for going with array over dict? I made an issue to revisit this: https://github.com/inveniosoftware/invenio-rdm-records/issues/78  

Schema as array:
https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/marshmallow/json.py#L61

Reply:
After talk: Going for dict and doing #78 with this one.